### PR TITLE
Removed lwip and updated to newest librs

### PR DESCRIPTION
**Pull request by CarlWachter**
hermit-os/kernel#656 can also be merged after this, since `hermit/mailbox.h` was removed.
hermit-os/kernel#1165 also depends on this PR.

See also: hermit-os/newlib#1

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,3 @@
 [submodule "librs"]
 	path = librs
-	url = https://github.com/hermit-os/kernel.git
-[submodule "lwip"]
-	path = lwip
-	url = https://github.com/hermit-os/LwIP.git
+	url = https://github.com/hermitcore/libhermit-rs.git
```
Comment (mkroening) [inline .gitmodules:3]:
```suggestion
	url = https://github.com/hermit-os/kernel.git
```

File: usr/tests/CMakeLists.txt [hunk 2]
```diff
@@ -21,9 +21,9 @@ add_executable(thr_hello thr_hello.c)
 target_compile_options(thr_hello PRIVATE -pthread)
 target_link_libraries(thr_hello pthread)
 
-add_executable(nweb23 nweb23.c)
-target_compile_options(nweb23 PRIVATE -pthread)
-target_link_libraries(nweb23 pthread)
+#add_executable(nweb23 nweb23.c)
+#target_compile_options(nweb23 PRIVATE -pthread)
+#target_link_libraries(nweb23 pthread)
```
Comment (mkroening) [inline usr/tests/CMakeLists.txt:26]: What's preventing this to work?

**Review by mkroening — CHANGES_REQUESTED** [review]
The code looks fine. It would be great if we did not lose Fortran support if possible.

File: usr/tests/test-malloc-mt.c [hunk 3]
```diff
@@ -10,11 +10,11 @@
 #endif
 
 #ifndef NUM_ITER
-#define NUM_ITER    10000
+#define NUM_ITER    1000
```
Comment (mkroening) [inline usr/tests/test-malloc-mt.c:13]: Why decrease this number?

File: usr/tests/test-malloc.c [hunk 4]
```diff
@@ -4,7 +4,7 @@
 #include <malloc.h>
 
 #ifndef NUM_ITER
-#define NUM_ITER    100000
+#define NUM_ITER    10000
```
Comment (mkroening) [inline usr/tests/test-malloc.c:7]: And this one?

**Review by mkroening — APPROVED** [review]

File: usr/tests/test-malloc-mt.c (hunk 3 above)
Comment (CarlWachter) [reply usr/tests/test-malloc-mt.c:13]: Just so it runs fine with less than 2 GB of memory

**Review by mkroening — APPROVED** [review]

**Pull request merged**